### DISCUSSION TITLE
ditching

### DISCUSSION THREAD

**Discussion by MicrosoftFlightsim** [flybywiresim/aircraft#2908] [Ideas]
Please can you add the feature of the ditching button functioning. Btw keep up the brilliant work. :)

**Comment by MapleNAV-TCASFAULT** [comment]
FBW is trying to reach study level, so it'll definitely be implemented. But I don't find much point of that button, as ATM there is close to no water physics in MSFS.

**Comment by GACSavannah** [comment]
Water physics may not be implemented yet, but operation of the ditching switch does change a number of system operations, and it would be neat to see those modeled.
